src/aiida_quantumespresso/workflows/protocols/pw/relax.yaml

```diff
@@ -11,8 +11,8 @@ default_inputs:
     base_final_scf:
         pw:
             parameters:
-                CELL:
-                    press_conv_thr: 0.5
+                CONTROL:
+                    calculation: scf
```

Comment:
Let's not ask questions how a pressure convergence condition was the only part of the "scf" step of the `PwRelaxWorkChain` protocol. 😅